### PR TITLE
Battery non blocking read, fix calibration and System Info redesign

### PR DESCRIPTION
**Pull request by joaquimorg**
- Battery voltage non blocking read.
- Battery calibration.
- System Info redesign.

IMG_20210328_163657

IMG_20210328_163704

### CODE REVIEW

File: src/components/battery/BatteryController.cpp [hunk 1]
```diff
@@ -1,49 +1,92 @@
 #include "BatteryController.h"
 #include <hal/nrf_gpio.h>
+#include <nrfx_saadc.h>
 #include <libraries/log/nrf_log.h>
 #include <algorithm>
+#include <math.h>
 
 using namespace Pinetime::Controllers;
 
+#define BATTERY_VOL NRF_SAADC_INPUT_AIN7   // 31 Analog
```
Comment (JF002) [inline src/components/battery/BatteryController.cpp:10]: I think these constants and variables could be declared as member variables of the class `Battery`

**Comment by JF002** [comment]
Waiting for #232 to avoid adding again conflicts on that PR :)

**Comment by JF002** [comment]
Thanks, the redesign of the app is really good looking!

**Comment by joaquimorg** [comment]
This code is in new PR, this is now obsolete.

**Comment by JF002** [comment]
> This code is in new PR, this is now obsolete.

I can't find it, is this in a PR you plan to create soon?

**Pull request closed without merging**